remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,20 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+)
+
+func TestFlattenMap(t *testing.T) {
+	testString := []byte(`{"name":"shoubhik","assignee":{"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
+	var nestedMap map[string]interface{}
+	err := json.Unmarshal(testString, &nestedMap)
+
+	if err != nil {
```

Comment:
It is for 2 reasons;

* Get used to _always_ check for errors
* Next time someone need to make a change in the json and introduce a format error by mistake they now spend n hours trying to figure out where the problem is actually coming from